### ISSUE TITLE
Question regarding kernel arguments

### ISSUE THREAD

**Issue by zaibon** [threefoldtech/0-core#82]
It is normal that if I boot a 0-OS with the kernel arguments `support` and `development`, then I can access 0-core over the support interface without authentication ?

**Comment by muhamadazmy** [comment]
People always mix `development` and `support` are even use them together. Although they are not designed to be used together 

From the docs
- `development` If set, start the redis-proxy allow direct client connections, also opening the required client ports. If not set, no direct client connections will be allowed
- `support` If set, gives unconditional access to our staff (threefold sysadmin) members. Please only add this if specially required

Basically means once `development` is set.  Accept client connection with no authentication.

On the other hand if `support` is set it's same as development but only accept client connection from `organisation` param AND `threefold.sysadmin` organisation.

Basically once u set development, it overrule support.
